### ISSUE TITLE
Bug - Switch - accessibility issues due to dynamic labeling

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#10645]
**Describe the problem**
Followup to https://github.com/patternfly/patternfly/issues/6347. tldr is that Switch is currently not accessible due to us allowing dynamic labels when the Switch is checked or not checked. This can cause confusion as the context will be different, e.g. if the label is "Enable option" and labelOff is "Disable option", when checked the switch's accessible name is "Enable option, checked" which conveys that clicking the switch will mean the option is disabled, but when unchecked the accessible name is "Disable option, unchecked", which conveys that clicking the switch will disable the option).

The label text should not be dynamic based on the switch's checked state.

**How do you reproduce the problem?**
Provide steps to reproduce. A [codesandbox](https://codesandbox.io/s/serverless-cherry-q9t3f ) demonstrating the problem is appreciated.

**Expected behavior**
A clear and concise description of the expected behavior.

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**

**Comment by nicolethoen** [comment]
closed by #10646
